### PR TITLE
Updated discord details

### PR DESCRIPTION
**Pull request by creyente1897**
Changes requested in pr #221 
#### Describe the changes you have made in this pr -
Added details to tell users that this is an unofficial discord channel link

### CODE REVIEW

**Comment by creyente1897** [comment]
@armudgal Can you please review

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Here are links to our:
 
 [Gitter Channel](https://gitter.im/CircuitVerse/community)
 
-[Discord Channel](https://discord.gg/95x8H7b)
+[Discord Channel](https://discord.gg/95x8H7b) - This is an unofficial fan communication channel. Thanks to [@jaybox](https://github.com/jbox144) for this initiative.
```
Comment (armudgal) [inline README.md]: I think it's `jbox1` and not `jaybox`

**Comment by armudgal** [comment]
Okay so currently the links are looking kinda plain and simple. It would be great if instead of links we can have badges. Gitter badge is included at top of readme and you can find a badge for discord [here](https://shields.io/category/chat)

**Comment by creyente1897** [comment]
@armudgal I have created the badge but it is showing widget disabled so if you could please enable the widgets that would be great 
https://github.com/badges/shields/issues/1659

**Comment by armudgal** [comment]
@jbox144 Can you enable widgest on Discord.

**Comment by satu0king** [comment]
I have enabled the widget, can you confirm if it is working?
image

**Comment by creyente1897** [comment]
@satu0king Thanks, its working :smile: 
Discord

**Pull request merged**